content/api/commands/selectfile.md

```diff
@@ -0,0 +1,276 @@
+| `force`                      | `false`                                                                        | Forces the action, disables [waiting for actionability](#Assertions).                                                                             |
+| `log`                        | `true`                                                                         | Displays the command in the [Command log](/guides/core-concepts/test-runner#Command-Log).                                                         |
+| `timeout`                    | [`defaultCommandTimeout`](/guides/references/configuration#Timeouts)           | Time to wait for `.selectFile()` to resolve before [timing out](#Timeouts).                                                                       |
+| `waitForAnimations`          | [`waitForAnimations`](/guides/references/configuration#Actionability)          | Whether to wait for elements to [finish animating](/guides/core-concepts/interacting-with-elements#Animations) before executing the command.      |
+
+### Yields [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Subject-Management)
+
+- `.selectFile()` yields the same subject it was given from the previous
+  command.
+
+### Action
+
+Depending on the `action` set in the `options` argument, `.selectFile()` can
+simulate two different user behaviors:
+
+#### `select` (default)
+
+By default, `selectFile()` runs in 'select' mode, mimicking a user selecting one
+or more files on an HTML5 input element. In this mode, the
+[subject](/guides/core-concepts/introduction-to-cypress#Subject-Management) must
+be a single `input` element with `type="file"`, or a `label` element connected
+to an input (either with its `for` attribute or by containing the input).
+
+#### `drag-drop`
+
+Setting the action to `drag-drop` changes the behavior of the command to instead
+mimic a user dragging files from the operating system into the browser, and
+dropping them over the selected subject. In this mode, the subject can be any
+DOM element or the `document` as a whole.
+
+## Examples
+
+### From a file on disk
+
+```javascript
+cy.get('input[type=file]').selectFile('path/to/file.json')
+cy.get('input[type=file]').selectFile('path/to/file.png')
+```
+
+If given a path, `.selectFile()` will search for the file relative to the
+project root and attach the file exactly as it exists on disk. This is the
+preferred way to work with files on disk, avoiding many encoding-related
+pitfalls.
+
+### On a hidden input
+
+```javascript
+cy.get('input[type=file]').selectFile('file.json', { force: true })
+```
+
+In many cases in modern applications, the underlying file input is hidden from
+view, and activated by a user clicking on a button. In these cases, you will
+need to tell Cypress to ignore its actionability checks and attach the file even
+though a user would not be able to directly activate the file input.
```

Comment:
```suggestion
In many cases in modern applications, the underlying file input is hidden from
view, and activated by a user clicking on a button. In these cases, you will
need to tell Cypress to ignore its actionability checks and upload the file even
though a user would not be able to directly activate the file input.
```

Reply:
Going to use 'select' rather than 'attach' here.